draft-wang-ppm-differential-privacy.md

```diff
@@ -884,65 +910,129 @@ in order to achieve various combinations of `(EPSILON, DELTA)`-DP.
         result = []
-        for i in range(len(agg_share)):
-            noise = noise_vec[i]
-            if noise < 0:
-                noise = Field.MODULUS + noise
-            result.append(agg_share[i] + Field(noise))
+        for (agg_share_i, noise_vec_i) in zip(agg_share, noise_vec):
+            if noise_vec_i < 0:
+                noise_vec_i = Field.MODULUS + noise_vec_i
+            result.append(agg_share_i + self.Field(noise_vec_i))
         return result
 
     def debias_agg_result(self,
-                          agg_result: AggregateResult,
-                          meas_count: Unsigned,
-                          min_batch_size: Unsigned,
-                          num_aggregators: Unsigned,
-                          ) -> DebiasedAggregateResult:
-        """
-        No debiasing.
-        """
+                          agg_result: AggResult,
+                          meas_count: int,
+                          ) -> DebiasedAggResult:
+        # TODO(junyechen1996): Interpret large unsigned integers as
+        # negative values or 0 properly. For now, directly return it,
+        # since we haven't fully implemented discrete Gaussian
+        # mechanism (#10).
         return agg_result
 ~~~
 
-> TODO(issue #10): replace `dgauss_mechanism` once we have concretely defined
-> it in {{discrete-gaussian}}.
+#### Utility
+
+We will demonstrate the utility of this policy in the table below in terms of
+the standard deviation `SIGMA` in discrete Gaussian needed to achieve various
+combinations of `(EPSILON, DELTA)`-DP, based on the open-sourced code in
+{{AGM}}.
+
+It's worth noting that if more than one Aggregator is honest, we will lose some
+utility because each Aggregator is independently adding noise. The standard
+deviation in the aggregate result thus becomes `SIGMA * sqrt(c)`, where `c` is
+the number of honest Aggregators. In the table below, the numbers in
+"Standard deviation" column assume `c = 2`. The numbers in
+"Standard deviation (OAOC)" column assume only one Aggregator is honest.
+
+| `EPSILON`       | `DELTA`     | Standard deviation    | Standard deviation (OAOC) |
+|:----------------|:------------|:----------------------|:--------------------------|
+| 0.317           | 1e-9        | 33.0788               | 23.3903                   |
+| 0.906           | 1e-9        | 12.0777               | 8.5402                    |
+| 1.528           | 1e-9        | 7.3403                | 5.1904                    |
+{: #histogram-aggregator-dp title="Utility of Pure Aggregator-DP for histogram use case."}
+
+### Conclusion
+
+We just demonstrated two simple policies that achieve the same level of
+`(EPSILON, DELTA)`-DP on histogram use case, under different trust models. They
+are comparable in terms of utility, but because the second policy with only
+Aggregator-DP requires all Aggregators to independently add noise to maintain
+OAOC trust model, we lose some utility when more than one Aggregator is honest.
```

Comment:
Good summary! However, drafts don't usually have Conclusion sections the way papers do. I think this is a useful thing to note, but let's put it before the policies somewhere.